app/web/template/how_should_replacement_attorneys_step_in.gohtml

```diff
@@ -3,7 +3,62 @@
 {{ define "main" }}
```

Comment:
![image](https://user-images.githubusercontent.com/146390/201957437-4016c81e-6659-448f-9388-0a0831b6faeb.png)

some of these need to be `trHtml`

Reply:
Doh, thanks for catching.